config/bootstrap.php

```diff
@@ -50,7 +50,7 @@
  * variables as required.
  *
  * It is HIGHLY discouraged to use a .env file in production, due to security risks
```

Comment:
**an** .env file ?

Reply:
a `dot env`... that's at least how I read it

Reply:
That's totally correct then and... a very interesting moment of insight into my perception of things  👍

Reply:
could also have cheated `use .env fileS` :sunglasses:

Reply:
👏